apps/prairielearn/src/components/Navbar.html.ts

```diff
@@ -785,8 +785,8 @@ function NavbarInstructor({
         aria-expanded="false"
         ${!authz_data.overrides
           ? html`
-              hx-get="/pl/navbar/course/${course.id}/switcher" hx-trigger="show.bs.dropdown once
-              delay:200ms" hx-target="#navbarDropdownMenuCourseAdmin"
+              hx-get="/pl/navbar/course/${course.id}/switcher" hx-trigger="mouseover once, focus
+              once, show.bs.dropdown once delay:200ms" hx-target="#navbarDropdownMenuCourseAdmin"
```

Comment:
My changes from https://github.com/PrairieLearn/PrairieLearn/pull/11511 will format this a little more nicely. I'm happy to land your PR first and resolve conflicts on my branch.